### PR TITLE
Map sudo_require_reauthentication to SLES-12-010113

### PR DESCRIPTION
**Pull request by teacup-on-rockingchair**
#### Description:

- Add SLES-12-010113 rule defintion

#### Rationale:

- Map rule sudo_require_reauthentication to SLES-12-010113 stig id
- add CCE id and NIST reference

### CODE REVIEW

**Comment by ggbecker** [comment]
@teacup-on-rockingchair please rebase this one

File: linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml [hunk 1]
```diff
@@ -26,11 +26,14 @@ severity: medium
...
+    cce@sle12: CCE-83231-1
     cce@sle15: CCE-85764-9
 
 references:
     disa: CCI-002038
     stigid@rhel8: RHEL-08-010384
+    nist: IA-11
+    stigid@sle12: SLES-12-010113
     stigid@sle15: SLES-15-020102
     srgid: SRG-OS-000373-GPOS-00156
```
Comment (vojtapolasek) [inline linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml:38]: Please also fix this line, thank you.
```suggestion
    srg: SRG-OS-000373-GPOS-00156
```

File: products/sle12/profiles/stig.profile [hunk 2]
```diff
@@ -246,7 +246,11 @@ selections:
     - sssd_offline_cred_expiration
     - sudo_remove_no_authenticate
     - sudo_remove_nopasswd
+<<<<<<< HEAD
```
Comment (vojtapolasek) [inline products/sle12/profiles/stig.profile:249]: Probably leftover from rebase, please fix it.

**Review by vojtapolasek — DISMISSED** [review]
Thank you for the update, but see review comments please.

File: products/sle12/profiles/stig.profile (hunk 2 above)
Comment (teacup-on-rockingchair) [reply products/sle12/profiles/stig.profile]: Thanks :bow: @vojtapolasek fixed that and rebase once again

File: linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml (hunk 1 above)
Comment (teacup-on-rockingchair) [reply linux_os/guide/system/software/sudo/sudo_require_reauthentication/rule.yml]: Nice one :+1: - missed that. Now should be ok

**Review by ggbecker — APPROVED** [review]
thanks @vojtapolasek for the review. I'm merging this one.

**Pull request merged**